### DISCUSSION TITLE
wire:navigate triggers event twice

### DISCUSSION THREAD

**Discussion by ronnieparalejas** [livewire/livewire#7128] [7. Bugs (archived)] [answered]
### Livewire version

v3.0.10

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

I only have simple code like `<a href="/dashboard" wire:navigate>Dashboard</a>` and the event triggers when you press(mousedown) the button for a second and release(mouseup). This doesn't happen when you release the button quickly or before the fetching finish in mousedown.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment, marked as answer]
We're in the process of investigating all wire:navigate issues and I have tested out your scenario. I believe this has been fixed so I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
